packages/components/src/components/fade-wrapper/fade-wrapper.tsx

```diff
@@ -1,7 +1,14 @@
-import PropTypes from 'prop-types';
-import React from 'react';
+import React, { ReactNode } from 'react';
 import posed, { PoseGroup } from 'react-pose';
 
+type TFadeWrapperProps = {
+    children: ReactNode;
+    is_visible: boolean;
+    keyname: string;
+    type: 'top' | 'bottom' | Omit<string, 'top' | 'buttom'>;
+    className?: string;
+};
```

Comment:
@iman-fs according to packages/components/stories/fade-wrapper/README.md, this component has only 1 required prop besides children which is is_visible 🙂:
```suggestion
type TFadeWrapperProps = {
    children: ReactNode;
    is_visible: boolean;
    keyname?: string;
    type?: 'top' | 'bottom';
    className?: string;
};
```

Reply:
@maryia-binary you're right. I will also add a default value to `is_visible` .
and about your question, git detected it as rename.